reana_commons/config.py

```diff
@@ -58,3 +63,19 @@
         'reana_job_controller.json')
 }
 """REANA Workflow Controller address."""
+
+K8S_MINIMUM_CAPACITY_CPU = 1
+"""Minimum allocatable cpu capacity in cluster to start new workflows."""
+
+K8S_MINIMUM_CAPACITY_MEMORY = 1000000
+"""Minimum allocatable memory capacity in cluster to start new workflows."""
+
+K8S_MINIMUM_CAPACITY_PODS = 109
+"""Minimum allocatable pod capacity in cluster to start new workflows."""
+
+K8S_MAXIMUM_CONCURRENT_JOBS = 1
+"""Upper limit on concurrent jobs running in the cluster."""
```

Comment:
Please check the BSM example and set the defaults very liberally so that almost any input would be accepted... One workflow can generate many jobs and the system may suffer, but we only need to protect ourselves from people starting ~N workflows at the same time.  

Also it would be good if these are easily user-settable, e.g. reana-cluster template variable taking precedence.

Reply:
With 10 concurrent jobs the BSM example ran on my machine ok. I agree it should be cluster configuration and not in reana-commons - to be improved.